kpatch-build/kpatch-build

```diff
@@ -649,6 +658,26 @@ module_name_string() {
 	echo "${1//[^a-zA-Z0-9_-]/-}" | cut -c 1-55
 }
 
+is_supported_distro(){
+    distro=$1
+    for each_distro in ${SUPPORTED_DISTRO[@]}; do
+        if [[ $distro == $each_distro ]]; then
+            return 1;
+        fi
+    done;
+    return 0
```

Comment:
The return values are backwards.  Bash functions return 0 for "true" and 1 for "false".  This only works because "anolis" != "Anolis" so the two bugs cancel each other out :-)

Reply:
Thank you for catching this bug....lol... I fix it in the new commit, please review it when you are free. Thanks~~